### DISCUSSION TITLE
Confluent Cloud TriggerAuthentication RC2

### DISCUSSION THREAD

**Discussion by iferca** [kedacore/keda#1299] [Q&A / Need Help] [answered]
I had a TriggerAuthentication working with KEDA 1.5 formed by: 
* username: someuser
* password: somepassword
* authmode : sasl_plaintext

Now I'm testing with KEDA 2.0.0-RC2 and I'm not sure how to replace sasl_ssl with one of the options available in documentation for the sasl field with my existing confluent configuration. I've tried with the three mentioned: plaintext, scram_sha256 or scram_sha512. I see this error in the operator: 

```
ERROR    scalehandler    Error getting scalers    {"object": {"apiVersion": "keda.sh/v1alpha1", "kind": "ScaledJob", "namespace": "default", "name": "test"}, "error": "error getting scaler for trigger #0: error creating kafka client: kafka: client has run out of available brokers to talk to (Is your cluster reachable?)"} 
```
Same cluster with version 1.5 works fine, if I trigger the job manually it can reach and consume from confluent cluster, etc. 

Any hint?

**Comment by zroubalik** [comment, marked as answer]
you can try `sasl: plaintext` (and specify `username` and `password`) and `tls: enable`

**Reply by iferca** [reply]
It was in the documentation :facepalm: read it several times yet still missed it

> tls: Optional. To enable SSL auth for Kafka, set this to enable. If not set, TLS for Kafka is not used.

Thanks

**Reply by zroubalik** [reply]
No worries :) I am glad it is working.
